database/sql/src/main/scala/cromwell/database/slick/WorkflowStoreSlickDatabase.scala

```diff
@@ -13,19 +15,31 @@ trait WorkflowStoreSlickDatabase extends WorkflowStoreSqlDatabase {
 
   import dataAccess.driver.api._
 
-  override def updateWorkflowsInState(updates: List[(String, String)])
-                                     (implicit ec: ExecutionContext): Future[Unit] = {
-    val updateQueries = updates.map({
-      case (oldState, newState) => dataAccess.workflowStateForWorkflowState(oldState).update(newState)
-    })
+  override def abortAllRunning()
+                              (implicit ec: ExecutionContext): Future[Unit] = {
+    val action = dataAccess
+      .workflowStateForWorkflowState(WorkflowStoreState.Running)
+      .update(WorkflowStoreState.Aborting)
 
-    val action = DBIO.sequence(updateQueries)
     runTransaction(action) void
   }
 
-  override def updateWorkflowState(workflowId: String, newWorkflowState: String)
-                                     (implicit ec: ExecutionContext): Future[Int] = {
-    val action = dataAccess.workflowStateForId(workflowId).update(newWorkflowState)
+  override def initializeRestartedFlags()
```

Comment:
It's not really clear what 'initialize' means, perhaps 'markRunningAndAbortingWorkflowsAsRestarting' or a less awkward equivalent of that